documentation/getting-started.md

```diff
@@ -1,64 +1,29 @@
 # Get started
 
-## Step 1: Install the barista-components and Angular CDK
+## TLDR;
 
-Install the Barista components library via npm or yarn:
+`ng add @dynatrace/barista-components`
 
-`npm install --save @dynatrace/barista-components @angular/cdk highcharts@6 @types/highcharts @dynatrace/barista-icons d3-scale@^3.0.0 d3-shape@^1.3.5`  
-or  
-`yarn add @dynatrace/barista-components @angular/cdk highcharts@6 @types/highcharts @dynatrace/barista-icons d3-scale@^3.0.0 d3-shape@^1.3.5`
+## Installation
 
-## Step 2: Animations
+To install our barista components use our ng-add schematics with
```

Comment:
I would remove "our" with "the" in this section. And I would uppercase "Barista".